android/build.gradle

```diff
@@ -48,5 +52,24 @@ allprojects {
             // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
             url("$rootDir/../node_modules/react-native/android")
         }
+        maven {
+            // Mapbox SDK requires authentication to download from Mapbox's private Maven repository.
+            url 'https://api.mapbox.com/downloads/v2/releases/maven'
+            authentication {
+                basic(BasicAuthentication)
+            }
+            credentials {
+                // 'mapbox' is the fixed username for Mapbox's Maven repository.
+                username = 'mapbox'
+
+                // The value for password is read from the 'MAPBOX_DOWNLOADS_TOKEN' gradle property. 
+                // Set this property in a gradle.properties file that is not committed to version control. 
+                // Recommended place is in the Gradle user home folder («USER_HOME»/.gradle/gradle.properties).
```

Comment:
NAB: Maybe reference the npm setup script here

Reply:
updated